File: Lista 7/L7_Q7.c
/*******************************************************************
*   Faça um programa que dada a idade de uma
*   pessoa, retorna uma das seguintes mensagens:
*   “Não pode nem votar e nem dirigir”, “Pode votar,
*   mas não pode dirigir”, “Pode votar e pode dirigir”.
*******************************************************************/
#include <stdio.h>

int main(void){
    int idade;

    printf("Informe a sua idade: ");
    scanf("%d", &idade);

    if(idade < 16)
        printf("Nao pode nem votar e nem dirigir");
    else if (idade < 18)
        printf("Pode votar, mas nao pode dirigir.");
    else
        printf("Pode votar e dirigir.");

    return 0;
    
}

File: 1_Período/Lista 7/Python/L7_Q7.py
idade = int(input("Informe a sua idade: "))

if idade < 16:
    print(f"Não pode nem votar e nem dirigir.")
elif idade < 18:
    print(f"Pode votar, mas não pode dirigir.")
else:
    print(f"Pode votar e dirigir.")